pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -134,7 +140,12 @@ public class PendingAckHandleImpl extends PendingAckHandleState implements Pendi
 
     public final RecoverTimeRecord recoverTime = new RecoverTimeRecord();
 
+    private final long pendingAckInitFailureBackoffInitialTimeInMs = 5000;
```

Comment:
This will be used by the init backoff delay time. We'd better change to a lower value by default? for example: 100ms?

Reply:
From the test logs

```
2023-11-08T23:33:52,637 - WARN  - [pulsar-transaction-executor-16-1:PendingAckHandleImpl@960] - [persistent://tnx/ns1/testUnloadSubscriptionWhenFailedInitPendingAck][subName2] Failed to init transaction pending ack handler. It will be retried in 5000 Ms
org.apache.bookkeeper.mledger.ManagedLedgerException: mock fail
```

Reply:
Fixed.